src/google/credential.rs

```diff
@@ -203,17 +214,20 @@ mod tests {
                 )),
             )],
             || {
-                RUNTIME.block_on(async {
```

Comment:
Was there any reason for this to be run inside a tokio runtime?

Reply:
They used to be async. Thanks for cleanup.